packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -188,13 +192,17 @@ export const Animation = (React.forwardRef<HTMLDivElement, AnimationProps>((prop
       {...unhandledProps}
       className={!isChildrenFunction ? cx(animationClasses, className, (child as any)?.props?.className) : ''}
     >
-      {isChildrenFunction
-        ? ({ state }) =>
-            (children as AnimationChildrenProp)({
-              classes: cx(animationClasses, className, (child as any)?.props?.className),
-              state,
-            })
-        : React.cloneElement(child, { className: cx(animationClasses, className, (child as any)?.props?.className) })}
+      {isChildrenFunction ? (
+        ({ state }) =>
+          (children as AnimationChildrenProp)({
+            classes: cx(animationClasses, className, (child as any)?.props?.className),
+            state,
+          })
+      ) : (
+        <Ref innerRef={mergedRef}>
```

Comment:
`Ref` should be a direct child of `Transtition` otherwise there is no way to get a ref when `children` are function.

Reply:
Right, I missed that :P .
I had a problem adding `Ref` as `Transition`'s direct child, because `Ref` has `React.Children.only` check, which throws when children is a function. So I did it a bit differently.
Also while testing, I found out `Animation` is still casted to `React.FC`, which does not accept `ref`. So I changed the typing as well.

Reply:
I also added another ref for `nodeRef`, because the returned result from `useMergedRefs` still has function type, and it won't pass proptype check :(